client/src/leap/soledad/client/sqlcipher.py

```diff
@@ -433,6 +433,7 @@ class SQLCipherU1DBSync(SQLCipherDatabase):
     same database replica.
     """
     syncing_lock = defaultdict(threading.Lock)
+    received_docs = []
```

Comment:
I use the idiom where we aren't implementing the constructor, to ensure the attribute exists, but in this case it's more appropiate to do it in the constructor. fixed.